java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_2/UserAccountValidator.java

```diff
@@ -0,0 +1,140 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class UserAccountValidator implements Validator {
+    IdUserValidator idUserValidator = new IdUserValidator();
```

Comment:
`The validator needs to use a list of smaller validators to perform a validation of specific account fields.`

Reply:
fixed